src/librustc_lint/builtin.rs

```diff
@@ -1286,10 +1286,24 @@ declare_lint! {
     "`...` range patterns are deprecated"
 }
 
-declare_lint_pass!(EllipsisInclusiveRangePatterns => [ELLIPSIS_INCLUSIVE_RANGE_PATTERNS]);
+pub struct EllipsisInclusiveRangePatterns {
+    node_id: Option<ast::NodeId>,
+}
+
+impl_lint_pass!(EllipsisInclusiveRangePatterns => [ELLIPSIS_INCLUSIVE_RANGE_PATTERNS]);
+
+impl EllipsisInclusiveRangePatterns {
```

Comment:
You can `#[derive(Default)]` for this instead.

Reply:
I chose it b/c of consistency, there's `new` on other linters, e.g. https://github.com/rust-lang/rust/blob/5160955b96636db3b3b70a348c53454e73473538/src/librustc_lint/builtin.rs#L552 and https://github.com/rust-lang/rust/blob/5160955b96636db3b3b70a348c53454e73473538/src/librustc_lint/lib.rs#L99 

`#[derive(Default)]` would give me `default` method

Reply:
@stepnivlk It seems preferable to use `#[derive(Default)]` wherever possible for these lints. Could you file an issue for converting all places in a different PR?